docs/source/github/github.rst

```diff
@@ -254,6 +254,55 @@ remote origin:
 Again, the pull request will be updated accordingly and CI checks
 will be re-triggered.
 
+Cherry-picking your PR to other branches
+----------------------------------------
+
+After your PR is merged into the master branch, you need to consider whether it should be backported to earlier branches.
+If the content is a new feature designated for the next release, obviously backporting is not appropriate. But if it is a fix or
+update to an existing topic, don't forget to cherry-pick the PR back to earlier branches as needed.
+You can use the GitHub cherry-pick command, or an easier option is to paste the following command as a comment in your PR after it is merged:
+
+.. code::
+
+   @Mergifyio backport release <x.y>
```

Comment:
release-<x.y>
Since you use the release-2.0 example below, might as well do it here as well.